models.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey, Boolean
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.ext.serializer import loads, dumps
+
+
+Base = declarative_base()
+
+
+class ModelRoom(Base):
+
+    __tablename__ = "rooms"
+    id = Column(Integer, primary_key=True)
+    name = Column(String(255), unique=True)
+    room_type = Column(String(1))
+    people_in_room = relationship(
+        "ModelPerson", back_populates="room", uselist=True)
+
+    def __repr__(self):
+        return "<Room (name='%s')>" % (self.name)
+
+    def to_obj(self):
+        from room_allocation import LivingSpace, Office, Fellow, Staff
```

Comment:
You can still avoid cyclic imports by including it at the top, so long as its after the other import. If they are in the right order, it will not cause cyclic imports